packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -160,7 +172,29 @@ export class Stepper implements LocalizedComponent, T9nComponent {
   render(): VNode {
     return (
       <Host aria-label={this.messages.label} role="region">
-        <slot onSlotchange={this.handleDefaultSlotChange} />
+        <div
+          class={{ container: true, [CSS.singleView]: this.singleViewMode }}
+          ref={this.setContainerEl}
+        >
+          {this.singleViewMode && (
+            <div class={{ [CSS.stepBarContainer]: true }}>
+              {this.items.map((item, index) => (
+                <StepBar
+                  disabled={item.disabled && index !== this.currentActivePosition}
+                  isActive={index === this.currentActivePosition}
+                  isComplete={item.complete && index !== this.currentActivePosition && !item.error}
+                  isError={item.error && index !== this.currentActivePosition}
```

Comment:
should this be `hasError` or `showError`  instead of `isError`?

Reply:
I think we can just use `error` since it is a UI state rather than a `boolean`.